WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -218,16 +218,18 @@ import WordPressComAnalytics
         let service = ReaderTopicService(managedObjectContext: ContextManager.sharedInstance().mainContext)
         service.siteTopicForSiteWithID(siteID!,
             isFeed:isFeed,
-            success: { [weak self] (objectID:NSManagedObjectID!, isFollowing:Bool) -> Void in
-                do {
-                    let context = ContextManager.sharedInstance().mainContext
-                    let topic = try context.existingObjectWithID(objectID) as? ReaderAbstractTopic
-                    self?.readerTopic = topic
-                } catch let error as NSError {
-                    DDLogSwift.logError(error.localizedDescription)
+            success: { [weak self] (objectID:NSManagedObjectID!, isFollowing:Bool) in
```

Comment:
I'd personally lose the type information here, as it can all be inferred (and drop `isFollowing`, as its value isn't being used):

`success: { [weak self] objectID, _ in`